.github/workflows/deploy-to-control-plane-review-app.yml

```
@@ -341,6 +359,8 @@ jobs:
     needs: build
     if: needs.build.outputs.do_deploy != 'false'
     runs-on: ubuntu-latest
+    env:
+      COMMENT_ID: ${{ needs.process-deployment.outputs.comment_id }}
```

actionlint (1.7.4)

property "process-deployment" is not defined in object type {build: {outputs: {image_tag: string}; result: string}}

(expression)